tests/unistd/dup.c

```diff
@@ -29,7 +57,6 @@ int main() {
   printf("close(f1): %d\n", close(f));
   printf("close(f2): %d\n", close(f2));
   printf("close(f3): %d\n", close(f3));
-  printf("\n");
```

Comment:
There is a bug in testing framework! this extra newline does not actually match to data file. But tests does not mention that.